scripts/make-release

```diff
@@ -164,7 +175,7 @@ if ! git branch -a | grep -q "remotes/upstream/${BRANCH}\$"; then
 else
     echo "You appear to already have a ${BRANCH} created, using it"
     git checkout "${BRANCH}"
-    git pull upstream "${BRANCH}"
+    git pull origin "${BRANCH}"
```

Comment:
@fsbraun how come you've changed the way this works?

Reply:
I had merged all changes for the release into my fork (`origin`) before running the script. I couldn’t merge them into upstream `release/3.11.x since` it’s protected.